geomstats/learning/radial_kernel_functions.py

```diff
@@ -0,0 +1,385 @@
+"""Radial kernel functions.
```

Comment:
This is great, thank you! I am just worrying that this type of .py file should belong to a machine learning library, and not really to a geometric machine learning library like geomstats. 

There is nothing specifically geometric about them. Isn't there any ML library that has these functions, such that we could just import them?

If not, (but probably not for now), you could also think about submitting a PR to scikit-learn ;)

Reply:
Or maybe following scipy.rbf? https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.Rbf.html